work-item-comments.go

```diff
@@ -55,8 +52,7 @@ func (c *WorkItemCommentsController) Create(ctx *app.CreateWorkItemCommentsConte
 
 		err = appl.Comments().Create(ctx, &newComment)
 		if err != nil {
-			jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrInternal(err.Error()))
-			return ctx.InternalServerError(jerrors)
+			return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
```

Comment:
@aslakknutsen IMHO this is more than enough: `return jsonapi.JSONErrorResponse(ctx, err)`. Except for Unauthorized errors, each repository does return proper errors that can be mapped to the correct error automatically. If no mapping can be done, we default to InternalServerError which is also fine.